src/synapse-plugins/converter-plugin/converter-data.vala

```diff
@@ -20,34 +20,47 @@
 */
 
 namespace Synapse {
-    const string MASS = "unit of mass";
-    const string LENGTH = "unit of length";
-    const string AREA = "unit of area";
-    const string VOLUME = "unit of volume";
-    const string TIME = "unit of time";
-    const string VELOCITY = "unit of velocity";
-
     const SIPrefix [] PREFIXES = {
-        {"yotta", "Y", 1E24},
-        {"zetta", "Z", 1E21},
-        {"exa", "E", 1E18},
-        {"peta", "P", 1E15},
-        {"tera", "T", 1E12},
-        {"giga", "G", 1E9},
-        {"mega", "M", 1E6},
-        {"kilo", "k", 1000},
-        {"hecto", "h", 100},
-        {"deca", "da", 10},
-        {"deci", "d", 0.1},
-        {"centi", "c", 0.01},
-        {"milli", "m", 0.001},
-        {"micro", "u", 1E-6},
-        {"nano", "n", 1E-9},
-        {"pico", "p", 1E-12},
-        {"femto", "f", 1E-15},
-        {"atto", "a", 1E-18},
-        {"zepto", "z", 1E-21},
-        {"yocto", "y", 1E-24}
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("yotta"), "Y", 1E24},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("zetta"), "Z", 1E21},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("exa"), "E", 1E18},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("peta"), "P", 1E15},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("tera"), "T", 1E12},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("giga"), "G", 1E9},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("mega"), "M", 1E6},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("kilo"), "k", 1000},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("hecto"), "h", 100},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("deca"), "da", 10},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("deci"), "d", 0.1},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("centi"), "c", 0.01},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("milli"), "m", 0.001},
+// TRANSLATORS: S.I. Unit Prefix
+        {N_("micro"), "u", 1E-6},
```

Comment:
```suggestion
        {N_("micro"), "µ", 1E-6},
```

Reply:
I use "u" (as does the `units` utility) for ease of entry in most locales (?). Ideally both abbreviations should be recognized I guess but atm there is no provision for multiple abbreviations.  It is not really possible to have localized entries as it would required a reverse translation to get a matchable string.

Reply:
Added extra prefix entry for "µ"